tests/good_deployments/template.yaml

```diff
@@ -9,33 +9,39 @@ deployment:
   description: ''
   intended_use: ''
   data_product_type: 'Summary statistics' # Fill in correct value.
-  data_product_region: ''
   publication_date: '1900-01-01' # Fill in correct value.
+  data_product_region: ''
+  data_product_sector: 'Technology' # Fill in correct value.
 
   # Tiers 2 and 3:
   dp_flavor:
     name: Pure DP # Fill in correct value.
+    input_metric: ''
+    bound_on_input_distance: ''
+    output_measure: ''
+    bound_on_output_distance: ''
     data_domain: '' # Tier 3
     unprotected_quantities: '' # Tier 3
 
   privacy_loss:
     privacy_unit: ''
     privacy_unit_description: ''
-    privacy_parameters: # Fill in correct value.
+    privacy_parameters: # Fill in correct value, and remove others
       epsilon: 1.0
-      # delta: ?
-      # rho: ?
+      delta: 0
```

Comment:
Could be useful to show how to express a delta in scientific notation.  I.e.,
`epsilon: 1.0e-07`